### ISSUE TITLE
Failed SMSes

### ISSUE THREAD

**Issue by tim-pearce** [tbar0970/jethro-pmm#1004]
I use ClickSend for sending SMSes but I expect the problem is the same with other services.
I notice some SMSes can fail some time after they were sent. Presumably they look valid at time of sending but the receiving system rejects them (most likely closed accounts) but this would take time.
Off hand, I don't know what the solution would be unless there is an API to request such failures, but then what would Jethro do with them?

**Comment by tbar0970** [comment]
Yep.  There are some errors that the SMS gateway will throw as soon as you submit the job to them.  Other errors will only be reported later, eg if they can't contact the handset after 48 hours or whatever. Jethro could query the gateway for updates after the event, and would then have to do something with the answers.

**Comment by tbar0970** [comment]
Related: https://github.com/tbar0970/jethro-pmm/pull/606
